modules/images/webp-uploads/fallback.js

```diff
@@ -4,21 +4,57 @@ window.wpPerfLab = window.wpPerfLab || {};
 	window.wpPerfLab.webpUploadsFallbackWebpImages = function( media ) {
 		for ( var i = 0; i < media.length; i++ ) {
 			try {
-				if ( ! media[i].media_details.sources || ! media[i].media_details.sources['image/jpeg'] ) {
-					continue;
-				}
-
-				var ext = media[i].media_details.sources['image/jpeg'].file.match( /\.\w+$/i );
-				if ( ! ext || ! ext[0] ) {
-					continue;
-				}
+				var image         = media[ i ],
+					media_details = image.media_details,
+					media_sources = media_details.sources,
```

Comment:
See my other feedback: Now this variable shouldn't even be relevant anymore, so let's remove it. Based on the REST API response updates, it won't ever be part of the data.